README.md

```diff
@@ -1,27 +1,43 @@
 
-- Install [node.js](https://nodejs.org/en/download/)
-- Install the [Vercel CLI](https://vercel.com/cli)
-- Clone the repo to your local development workspace
-- Run `npm install` to install dependencies
+## The Tech
+The Sitecore developer portal is built with Next.js, Typescript, Tailwind CSS, and is hosted on Vercel. The app uses static site generation to create all the pages at build time. It also utilizes Incremental Static Regeneration (ISR) to automatically update the app when changes to page content are made. Much of the page content is written in Markdown and is converted to HTML at build time.
+
+## Build Prerequisites
+
+### Node.js
+
+The developer portal is built with Next.js, so you'll need to have Node.js installed to build the project. You can find the latest version of Node.js [here](https://nodejs.org/en/). We recommend using the LTS version of Node.js.
+
+### Environment Variables
+The Sitecore developer portal incorporates a number of third party services to bring in content. For full functionality, you must create a **.env.local** file in the root of the project and add in the below environment variables. 
 
-## Getting Started
 
-First, run the development server:
+The following variables should exist within the .env.local file: 
 
-```bash
-npm run dev
-# or
-yarn dev
+```
+YOUTUBE_API_KEY="An API key with YouTube Data API v3 access enabled"
+TWITTER_BEARER_TOKEN="A bearer token from Twitter "
 ```
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+*Note: The site will still function without the above keys. The components that require these environment variables will fail gracefully and not display on the pages.*
 
-You can start editing the page by modifying `pages/index.js`. The page auto-updates as you edit the file.
+## Getting Started
+1. Install [Node.js](htts://nodejs.org/en/), we recommend the LTS version.
+2. Clone the repository.
+3. Inside the repository run `npm install` to install all the dependencies.
+4. Create a `.env.local` file in the root of the project and add the following environment variables:
+```
+YOUTUBE_API_KEY=""
+TWITTER_BEARER_TOKEN=""
+```
+ (For more information on populating environment variables see section **Environment Variables** above.)
 
-[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.js`.
+ 5. Run `npm run dev` to start the development server.
+ 6. Open the **http://localhost:3000** in your browser to see the result!
 
-The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
+## Bugs and Enhancements 
+We are actively working on improving the developer portal. If you have any suggestions or issues, please let us know. Opening an issue on GitHub is a great way to get started.
```

Comment:
We probably want to include a CONTRIBUTING & CODE_OF_CONDUCT adding to the repo, that can be linked to here?

The ones from the JSS repo would work well for now?

Reply:
Good idea. The Code of Conduct looks easy to link to for now. The contributing guide I'll pull in the pieces that make sense for this project. It will grow as we see what mistakes or hiccups are common with outside contributors.